### PR TITLE
Remove `--jobinfo`, standalone `config.yml` file

### PR DESCRIPTION
**Pull request by abhidg**
- docs: link to generated Sphinx documentation #87
- docs: remove --jobinfo and create standalone config.yml

### CODE REVIEW

File: cats/__init__.py [hunk 1]
```diff
@@ -48,24 +54,41 @@ def parse_arguments():
...
+    carbon intensity of calculations on smaller computers. For example, the
+    above calculation can be scheduled by running:
 
         cats -d 90 --loc OX1 -s at -c 'mycommand'
+
+    To report carbon footprint, pass the `--config` option to select a
+    configuration file and the `--profile` option to select a profile. An
+    example config file is given below:
+
+{config_text}
```
Comment (tlestang) [inline cats/__init__.py:81]: Very neat. Is that some sort of sphinx include syntax?

File: cats/__init__.py [hunk 2]
```diff
@@ -48,24 +54,41 @@ def parse_arguments():
...
+    Examples
+    ********
 
-    Cats can be used to report information on the best time to run a calculation and the amount
-    of CO2. Information about a 90 minute calculation in centeral Oxford can be found by running:
+    CATS can be used to report information on the best time to run a calculation
+    and the amount of CO2. Information about a 90 minute calculation in centeral
+    Oxford can be found by running:
 
-        cats -d 90 --loc OX1 --jobinfo="cpus=2,gpus=0,memory=8,partition=CPU_partition"
```
Comment (tlestang) [inline cats/__init__.py:58]: Oops, I forgot to remove this reference to the old 'jobinfo' syntax. Good catch :)

File: pyproject.toml [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 [tool.setuptools]
   packages = ["cats"]
+  package-data.cats = ["config.yml"]
```
Comment (tlestang) [inline pyproject.toml:7]: I think `config.yml` would end up in `lib/python3.X/site-packages/cats` ? Might not be straightforward to use it from there?

**Review by tlestang — APPROVED** [review]

File: cats/__init__.py (hunk 1 above)
Comment (abhidg) [reply cats/__init__.py:81]: Just the standard f-string with the file read in

**Pull request merged**